bigtop-packages/src/rpm/phoenix/SPECS/phoenix.spec

```diff
@@ -92,6 +92,12 @@ Requires: bsh-utils
 Requires: coreutils
 %endif
 
+%if  0%{?rocky} >= 8
```

Comment:
If we can cover RHEL 8 too, it would be nice. @yamasakisua

Reply:
@iwasakims 
Thanks comment.
Just an idea, change condition to include RHEL8 like this.

`%if 0%{?rhel} >= 8`

but this condition will include all RHEL distro (RHEL, CentOS, Rocky etc..).
Is it acceptable to include all RHEL distros?
Or change it to include only RockyLinux8 and RHEL8?

Reply:
I think the issue is reproducible in other RHEL 8 variant too. While only Rocky Linux 8 is officially supported by Bigtop, RPMs are usually usable on RHEL/CentOS too without modification. We have no gain by limiting the scope here. @yamasakisua

Reply:
@iwasakims 
I got it.
I change code to adopt all RHEL distro and push it.

Reply:
I have done the same test as here.
https://github.com/apache/bigtop/pull/1045#issue-1422543597